docs/building-on-lisk/deploying-smart-contract/with-Hardhat.md

```diff
@@ -207,6 +207,66 @@ contract NFT is ERC721 {
 }
 ```
 
+## Testing the contract
+To test our contract, we are going to use [Hardhat Network](https://hardhat.org/hardhat-network/docs/overview), a local Ethereum network designed for development.
+It comes built-in with Hardhat, and it's used as the default network.
+You don't need to setup anything to use it.
+
+In our tests we're going to use [ethers.js](https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-ethers) to interact with the NFT contract we built in the previous section, and we'll use [Mocha](https://mochajs.org/) as our test runner.
```

Comment:
May be we can update the section to be point based to make it structured. May be something like:
 
```suggestion
To test our NFT contract, we’ll use:
- **Hardhat Network** – A local Ethereum network for development. It comes built-in with Hardhat, and it's used as the default network.
- **Ethers.js** – To interact with the contract.
- **Mocha** – For test runner and assertions.
```